### PR TITLE
Add foresight openapi schema

### PR DESCRIPTION
**Pull request by forefab**

### CODE REVIEW

File: api_v1.yaml [hunk 1]
```diff
@@ -0,0 +1,904 @@
+openapi: 3.0.0
+info:
+  title: User and Customer API
```
Comment (apanakkat) [inline api_v1.yaml:3]: Since this public, could you add some more information here?

Comment (apanakkat) [reply api_v1.yaml:3]: Maybe a better title that this refers to the foresight api and also some description for the user might be useful.

**Review by apanakkat — APPROVED** [review]
Thanks!

**Comment by apanakkat** [comment]
A better description for the PR would be useful. Especially since this is a public repo.

**Review by forefab — COMMENTED** [review]
PTAL

**Comment by apanakkat** [comment]
Thanks a lot. LGTM.

**Pull request merged**